compiler/rustc_session/src/options.rs

```diff
@@ -532,6 +534,18 @@ macro_rules! options {
             true
         }
 
+        fn parse_debuginfo(slot: &mut DebugInfo, v: Option<&str>) -> bool {
+            match v {
+                Some("0") | Some("none") => *slot = DebugInfo::None,
+                Some("line-directives-only") => *slot = DebugInfo::LineDirectivesOnly,
```

Comment:
I would perhaps opt to omit the `-only` here.  In particular because in the future it may make sense to combine these with other kinds of debug-info in the future and e.g. `-Cdebuginfo=something-only,someotherthing` would seem weird.

Reply:
I choose the `only` suffix to follow the convention set by llvm and clang. These options are also mutually exclusive and disable any additional debug-information from being generated by llvm, so they wouldn't work with future additional options. For future combination options I think something like `-Cdebuginfo=limited,+additonal,-unwanted` would be probably be best.